src/adapters/EOFeedAdapter.sol

```diff
@@ -177,6 +183,14 @@ contract EOFeedAdapter is IEOFeedAdapter, Initializable {
         return priceData.eoracleBlockNumber;
     }
 
+    function _normalizePrice(uint256 price) internal view returns (int256) {
+        if (_inputDecimals >= _outputDecimals) {
+            return int256(price / (10 ** (_inputDecimals - _outputDecimals)));
+        } else {
+            return int256(price * (10 ** (_outputDecimals - _inputDecimals)));
+        }
+    }
+
     // slither-disable-next-line unused-state,naming-convention
     // solhint-disable-next-line ordering
     uint256[50] private __gap;
```

Comment:
need to reduce by 1

Reply:
this really makes sense only if this class is inherited. not sure it makes sense

Reply:
As we are not upgrading a contract, we can deploy these with 50 gap.
But I do think we need a gap here for future needs